tests/TextDiffTest.php

```diff
@@ -90,7 +90,7 @@ public function testAllFormatsCovered()
         sort($formats);
 
         $this->assertSame(
-            array('json', 'txt', 'xml'),
+            array('json', 'junit', 'txt', 'xml'),
```

Comment:
please add `junit` at the array at line 71 (https://github.com/bankiru/PHP-CS-Fixer/blob/9f2fd37f6f2e8e9eaf495c29ced2548350634106/tests/TextDiffTest.php#L71) as well to test the output (escaping)